### PR TITLE
[class.mem, class.mfct.non-static] Convert nonstatic to non-static si…

### PR DESCRIPTION
**Pull request by AaronBallman**
…nce the hyphenated use is the more common term.

### CODE REVIEW

File: source/classes.tex [hunk 1]
```diff
@@ -856,8 +856,8 @@
 Also see~\ref{temp.arg}.
 \exitnote
 
-\rSec2[class.mfct.non-static]{Nonstatic member functions}%
-\indextext{member function!nonstatic}
+\rSec2[class.mfct.non-static]{Non-static member functions}%
+\indextext{member function!non-static}
 
 \pnum
 A \grammarterm{non-static} member function may be called for an object of
```
Comment (zygoloid) [inline source/classes.tex]: Can you remove the bogus \grammarterm while you're here?

**Comment by AaronBallman** [comment]
I've removed the bogus grammar term. I didn't squash because they seem like unrelated changes, but I can try to squash if you would really like me to. :-P

**Comment by tkoeppe** [comment]
No, the two commits are fine now.

Does this change require regenerating the grammar and xrefs?

**Comment by AaronBallman** [comment]
When I ran latexmk, it did say that it was generating a file for
grammarindex.idx, so I would imagine it does, but am not 100% certain (I'm
not overly familiar with latexmk).

**Comment by tkoeppe** [comment]
There are "regenerate" guides in the Readme.

**Comment by AaronBallman** [comment]
Thank you for the hint about regenerating from the readme, there were indeed modifications which I've added.

**Comment by tkoeppe** [comment]
The way `makegram` works depends on the particular flavour of `sed` on your system. For now, can you edit the grammar changes so that they only contain the relevant diff?

**Comment by tkoeppe** [comment]
Richard has fixed something, can you fetch and rebase? Then the makegram script should work better.

**Comment by AaronBallman** [comment]
I fetched and rebased, which seems to have removed all the inserted whitespace from f586dad.

**Comment by tkoeppe** [comment]
My bad, the change doesn't affect the grammar at all, so no need to regenerate anything. As you were.

**Pull request merged**